### PR TITLE
[New Rules] [BBR] Windows Deprecated ERs Conversion - 2

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

https://github.com/elastic/ia-trade-team/issues/166

## Summary

Adds the following rules:

* Execution via MS VisualStudio Pre/Post Build Events
* Suspicious Troubleshooting Pack Cabinet Execution
* Windows Installer with Suspicious Properties
* Suspicious Execution via MSIEXEC
* Unsigned BITS Service Client Process

### CODE REVIEW

File: rules_building_block/defense_evasion_execution_via_visualstudio_prebuildevent.toml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+id = "TA0005"
+name = "Defense Evasion"
+reference = "https://attack.mitre.org/tactics/TA0005/"
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+
+[rule.threat.tactic]
+id = "TA0002"
+name = "Execution"
+reference = "https://attack.mitre.org/tactics/TA0002/"
```
Comment (Aegrah) [inline rules_building_block/defense_evasion_execution_via_visualstudio_prebuildevent.toml:96]:
```suggestion
reference = "https://attack.mitre.org/tactics/TA0002/"

```

File: rules_building_block/defense_evasion_msdt_suspicious_diagcab.toml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  [[rule.threat.technique]]
+  name = "System Binary Proxy Execution"
+  id = "T1218"
+  reference = "https://attack.mitre.org/techniques/T1218/"
+
+  [rule.threat.tactic]
+  name = "Defense Evasion"
+  id = "TA0005"
+  reference = "https://attack.mitre.org/tactics/TA0005/"
```
Comment (Aegrah) [inline rules_building_block/defense_evasion_msdt_suspicious_diagcab.toml:60]:
```suggestion
  reference = "https://attack.mitre.org/tactics/TA0005/"
  
```

File: rules_building_block/defense_evasion_msiexec_installsource_archive_file.toml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+id = "T1218.007"
+name = "Msiexec"
+reference = "https://attack.mitre.org/techniques/T1218/007/"
+
+
+
+[rule.threat.tactic]
+id = "TA0005"
+name = "Defense Evasion"
+reference = "https://attack.mitre.org/tactics/TA0005/"
```
Comment (Aegrah) [inline rules_building_block/defense_evasion_msiexec_installsource_archive_file.toml:65]:
```suggestion
reference = "https://attack.mitre.org/tactics/TA0005/"

```

File: rules_building_block/defense_evasion_suspicious_msiexec_execution.toml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/09/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious execution of the built-in Windows Installer, MSIEXEC, to install a package from usual paths or
```
Comment (Aegrah) [inline rules_building_block/defense_evasion_suspicious_msiexec_execution.toml:12]:
```suggestion
Identifies suspicious execution of the built-in Windows Installer, msiexec.exe, to install a package from usual paths or
```

File: rules_building_block/defense_evasion_suspicious_msiexec_execution.toml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+references = [
+  "https://lolbas-project.github.io/lolbas/Binaries/Msiexec/",
+  "https://www.guardicore.com/labs/purple-fox-rootkit-now-propagates-as-a-worm/"
+]
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Suspicious Execution via MSIEXEC"
```
Comment (Aegrah) [inline rules_building_block/defense_evasion_suspicious_msiexec_execution.toml:24]:
```suggestion
name = "Suspicious Execution via msiexec.exe"
```

File: rules_building_block/defense_evasion_suspicious_msiexec_execution.toml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+id = "T1218.007"
+name = "Msiexec"
+reference = "https://attack.mitre.org/techniques/T1218/007/"
+
+
+
+[rule.threat.tactic]
+id = "TA0005"
+name = "Defense Evasion"
+reference = "https://attack.mitre.org/tactics/TA0005/"
```
Comment (Aegrah) [inline rules_building_block/defense_evasion_suspicious_msiexec_execution.toml:81]:
```suggestion
reference = "https://attack.mitre.org/tactics/TA0005/"

```

File: rules_building_block/defense_evasion_unsigned_bits_client.toml [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+rule_id = "9a3884d0-282d-45ea-86ce-b9c81100f026"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Rule Type: BBR", "Data Source: Elastic Defend"]
+timestamp_override = "event.ingested"
+building_block_type = "default"
+type = "eql"
+
+query = '''
+library where dll.name : "Bitsproxy.dll" and process.executable != null and
+not process.code_signature.trusted == true and not user.id : "S-1-5-18"
```
Comment (Aegrah) [inline rules_building_block/defense_evasion_unsigned_bits_client.toml:35]: As this is defense evasion, are we sure we want to exclude the SYSTEM user?

File: rules_building_block/defense_evasion_unsigned_bits_client.toml [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+[[rule.threat.technique]]
+id = "T1197"
+name = "BITS Jobs"
+reference = "https://attack.mitre.org/techniques/T1197/"
+
+
+[rule.threat.tactic]
+id = "TA0005"
+name = "Defense Evasion"
+reference = "https://attack.mitre.org/tactics/TA0005/"
```
Comment (Aegrah) [inline rules_building_block/defense_evasion_unsigned_bits_client.toml:50]:
```suggestion
reference = "https://attack.mitre.org/tactics/TA0005/"

```

**Review by Aegrah — APPROVED** [review]
LGTM!

File: rules_building_block/defense_evasion_execution_via_visualstudio_prebuildevent.toml (hunk 1 above)
Comment (w0rk3r) [reply rules_building_block/defense_evasion_execution_via_visualstudio_prebuildevent.toml:96]: Same as https://github.com/elastic/detection-rules/pull/3131#discussion_r1340101958

File: rules_building_block/defense_evasion_execution_via_visualstudio_prebuildevent.toml [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Execution via MS VisualStudio Pre/Post Build Events"
+risk_score = 21
+rule_id = "fec7ccb7-6ed9-4f98-93ab-d6b366b063a0"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Rule Type: BBR", "Data Source: Elastic Defend"]
```
Comment (imays11) [inline rules_building_block/defense_evasion_execution_via_visualstudio_prebuildevent.toml:30]:
```suggestion
tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Tactic: Execution", "Rule Type: BBR", "Data Source: Elastic Defend"]
```

**Review by imays11 — APPROVED** [review]

**Pull request merged**